.github/workflows/ci.yml

```diff
@@ -68,13 +68,10 @@ jobs:
   linux:
     name: Build and Test on Linux
     runs-on: ubuntu-latest
+    container: swift:5.9
```

Comment:
following the advice [here](https://github.com/swift-actions/setup-swift/issues/591#issuecomment-1685710678) to avoid [this kind of CI failure](https://github.com/dfed/SafeDI/actions/runs/7581297537/job/20648662318)